### DISCUSSION TITLE
Grouped Albums in queue view.

### DISCUSSION THREAD

**Discussion by ksz16** [jcorporation/myMPD#1127] [Q&A] [answered]
Hi, new user here. I've been using myMPD for a few days and so far I have a very positive feeling. I am missing only one thing. I mean the ability to change the queue view to one in which songs are grouped by album. Cantata has something like that in its settings and you can choose between two styles: Grouped Albums and Table. Perhaps this feature exists in myMPD, but I can't find it, although I have read the documentation and searched on GitHub. Could someone please clarify this for me? Thank you in advance.

**Comment by jcorporation** [comment, marked as answer]
myMPD supports sorting the queue by tags, if MPD supports it. This is a new feature of MPD 0.24 (not yet released).

**Comment by ksz16** [comment]
Great to hear it - I'll be waiting impatiently for the new release. Thank you very much for your quick response and for all your work on this cool app.
